content/docs/ui/managing-contacts/segmenting-your-contacts.md

```diff
@@ -431,11 +434,33 @@ Operators available on a “Single Send” field include:
   </tr>
   <tr>
     <td>has clicked</td>
-    <td>Any contacts who clicked a Single Send</td>
+    <td>Any contact who has clicked a Single Send</td>
   </tr>
   <tr>
     <td>has opened</td>
-    <td>Any contacts who opened a Single Send</td>
+    <td>Any contact who has opened a Single Send</td>
+  </tr>
+</table>
+
+### Operators available for Automation Fields
+
+Automation conditions will filter contacts who have engaged with a particular Automation or any Automation based on the constraints of the condition.
+
+<table>
+  <tr>
+    <th>Operator Name</a></th>
+    <th>Description</a></th>
+  </tr>
+  <tr>
+    <td>was sent</td>
+    <td>Any contacts who were sent an Automation</td>
+  </tr>
+    <td>has opened</td>
+    <td>Any contacts who has opened an Automation</td>
+  </tr>
+  <tr>
+    <td>has clicked</td>
+    <td>Any contacts who has clicked an Automation</td>
```

Comment:
Since these apply to "any auto" or "specific auto" should we call that out? E.g., "Any contacts who were sent An Automation or a Specific Automation within the specified time range"

Also, should we add the same section for Single Send fields or augment this to have both?